File: app/src/main/java/yangchengyu/shmtu/edu/cn/calligraphyrecognize/utils/CompareClass.java
package yangchengyu.shmtu.edu.cn.calligraphyrecognize.utils;

import java.util.Comparator;

import yangchengyu.shmtu.edu.cn.calligraphyrecognize.bean.KNNDistance;

public class CompareClass implements Comparator<KNNDistance> {
    @Override
    public int compare(KNNDistance o1, KNNDistance o2) {
        return o1.getDisatance() > o2.getDisatance() ? 20 : -1;
    }
}

File: app/src/main/java/yangchengyu/shmtu/edu/cn/calligraphyrecognize/utils/CompareClass.kt
package yangchengyu.shmtu.edu.cn.calligraphyrecognize.utils

import yangchengyu.shmtu.edu.cn.calligraphyrecognize.bean.KNNDistance
import java.util.*

class CompareClass : Comparator<KNNDistance> {
    override fun compare(o1: KNNDistance, o2: KNNDistance): Int {
        return if (o1.distance > o2.distance) 20 else -1
    }
}
